components/logs-table/types.ts

```
@@ -12,17 +14,48 @@ export type UserReportedError = {
-  errors: string | null
+  diarization_v2: boolean
+  duration: number
   ended_at: string | null
+  errors: string | null
+  event_id: number | null
+  id: number
+  meeting_url: string
+  mp4_s3_path: string
+  params: {
+    bot_name: string
+    extra: Record<string, any>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)